### ISSUE TITLE
matrix-sdk version 0.7.0 breaks dependency

### ISSUE THREAD

**Issue by nadestin** [matrix-org/matrix-rust-sdk#3039]
upgrading cargo.toml matrix-sdk dependency from "0.6" to "0.7" breaks dependency:

$ cargo update
    Updating crates.io index
error: failed to select a version for the requirement `aquamarine = "^0.4.0"`
candidate versions found which didn't match: 0.5.0, 0.3.3, 0.3.2, ...
location searched: crates.io index
required by package `matrix-sdk v0.7.0`
    ... which satisfies dependency `matrix-sdk = "^0.7"` of package `matrix-connector v0.1.1 (<concealed>/matrix-connector)`
perhaps a crate was updated and forgotten to be re-vendored?

Searching crates.io for crate aquamarine shows that Version 0.4.0 had been removed about 1 Month ago by Mike Lubinets.

**Comment by bnjbvr** [comment]
Thanks for opening an issue. Yeah that's because the `aquamarine` dependency at version 0.4.0 has been yanked, so now we have to release a patch version for a few crates, that bumps this dependency's version :| See also [this issue](https://github.com/mersinvald/aquamarine/issues/48).

**Comment by bertybuttface** [comment]
Will there be a 0.7.1 release with updated dependencies?

**Comment by jplatte** [comment]
Either that, or the author of aquamarine un-yanks 0.4.0.

**Comment by bertybuttface** [comment]
So as a workaround we need to pin to this commit: https://github.com/matrix-org/matrix-rust-sdk/commit/147f1d3f6c11809ae9425d2ee9c21691c90899be instead of pinning to 0.7.0

E.g. 

```
matrix-sdk = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "147f1d3f6c11809ae9425d2ee9c21691c90899be" }
```

Is that correct?

**Comment by jplatte** [comment]
Sounds like a reasonable workaround, yes.

**Comment by bnjbvr** [comment]
I've now released a new version of `matrix-sdk` 0.7.1 that includes the bump, and I could compile a toy project with it. Closing, please reopen if there are still issues with it.
